examples/eks-cluster-with-new-vpc/main.tf

```diff
@@ -121,26 +121,7 @@ module "aws-eks-accelerator-for-terraform" {
       subnet_ids      = module.aws_vpc.private_subnets
     }
   }
-
-  # FARGATE
-  fargate_profiles = {
-    default = {
-      fargate_profile_name = "default"
-      fargate_profile_namespaces = [
-        {
-          namespace = "default"
-          k8s_labels = {
-            Environment = "preprod"
-            Zone        = "dev"
-            env         = "fargate"
-          }
-      }]
-      subnet_ids = module.aws_vpc.private_subnets
-      additional_tags = {
-        ExtraTag = "Fargate"
-      }
-    },
-  }
+  depends_on = [module.aws_vpc]
```

Comment:
I think we should avoid adding module level dependencies. Implicit dependencies should handle. Moreover, Customers should never try to create VPC and EKS Cluster in one apply. This may be ok for sandbox deployment for dev purposes

Reply:
If this is not increasing the TF PLAN time and helping in Destroy then we can keep it